level-ws.js

```diff
@@ -124,15 +112,14 @@ WriteStream.prototype._flush = function (f) {
     }
   }
 }
-
 WriteStream.prototype.toString = function () {
   return 'LevelUP.WriteStream'
 }
 
 WriteStream.prototype.destroy = function () {
-  if (this._destroyed) return
+  if (this.destroyed) return
   this._buffer = null
-  this._destroyed = true
+  this.destroyed = true
   this.writable = false
   this.emit('close')
 }
```

Comment:
I really wanted to use the inherited version of `.destroy()`, which is almost identical, with the difference of setting `.writable` to `false`.